archery/settings.py

```diff
@@ -486,6 +486,9 @@
     },
 }
 
+# 在网站标题及登录页面追加此内容, 可用于多archery实例的区分
+CUSTOM_TITLE_SUFFIX = ""
```

Comment:
```suggestion
CUSTOM_TITLE_SUFFIX = env("CUSTOM_TITLE_SUFFIX", “”)
```
帮忙测试下，这样应该就可以用环境变量，然后注释里加一行

Archery 配置后台也有相同配置，如都做了配置，以后台配置为准

Reply:
已修改。在.env文件添加配置已测试通过。

src\docker-compose\.env docker文件加上了。